runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -168,38 +220,104 @@ public final class HttpClientFactory implements HttpStreamFactory
 
     private final AsciiSequenceView asciiRO = new AsciiSequenceView();
 
-    private final HttpClientDecoder decodeHeaders = this::decodeHeaders;
-    private final HttpClientDecoder decodeHeadersOnly = this::decodeHeadersOnly;
-    private final HttpClientDecoder decodeChunkHeader = this::decodeChunkHeader;
-    private final HttpClientDecoder decodeChunkBody = this::decodeChunkBody;
-    private final HttpClientDecoder decodeChunkEnd = this::decodeChunkEnd;
-    private final HttpClientDecoder decodeContent = this::decodeContent;
-    private final HttpClientDecoder decodeTrailers = this::decodeTrailers;
-    private final HttpClientDecoder decodeEmptyLines = this::decodeEmptyLines;
-    private final HttpClientDecoder decodeUpgraded = this::decodeUpgraded;
-    private final HttpClientDecoder decodeIgnore = this::decodeIgnore;
+    private final HttpClientDecoder decodeHeaders = this::decodeHttp1Headers;
+    private final HttpClientDecoder decodeHeadersOnly = this::decodeHttp1HeadersOnly;
+    private final HttpClientDecoder decodeChunkHeader = this::decodeHttp1ChunkHeader;
+    private final HttpClientDecoder decodeChunkBody = this::decodeHttp1ChunkBody;
+    private final HttpClientDecoder decodeChunkEnd = this::decodeHttp1ChunkEnd;
+    private final HttpClientDecoder decodeContent = this::decodeHttp1Content;
+    private final HttpClientDecoder decodeTrailers = this::decodeHttp1Trailers;
+    private final HttpClientDecoder decodeEmptyLines = this::decodeHttp1EmptyLines;
+    private final HttpClientDecoder decodeUpgraded = this::decodeHttpUpgraded;
+    private final HttpClientDecoder decodeIgnore = this::decodeHttp1Ignore;
```

Comment:
Let's rename these to `decodeHttp11*` for both the field names and the method names.